docs/documentation/maya/simple_setup.md

```diff
@@ -1,221 +1,297 @@
+In case the fiber or its color has to be manipulated, go to the Attribute Editor, in the debug submenu, and customize the color, width and length of the drawn lines.
 
 <figure>
   <img src="images/setup_ribbon_muscle_7.png"> 
-  <figcaption>Figure: Connection Editor tool.</figcaption>
+  <figcaption>Figure 23 : Muscle fibers combing.</figcaption>
 </figure>
 
-If we now hit play we should see the muscle propperly attached and with the expected dynamics. However, it still won't respond to external inputs like, for example, flexing an arm.
+Pressing play should show the muscle properly attached and with the expected dynamics. However, it still won't respond to external inputs like, for example, flexing an arm which add rigidity to the muscle.
 
 ### 3. Connecting AdnSensors
 
 The process to connect and AdnSensor to an AdnRibbonMuscle is the exact same to the one followed [for AdnMuscle](#3-connecting-adnsensors).
 
 <figure>
   <img src="images/setup_ribbon_muscle_8.png"> 
-  <figcaption>Figure: Connection Editor tool.</figcaption>
+  <figcaption>Figure 24: Connection Editor tool with AdnRotation sensor connected to AdnRibbonMuscle.</figcaption>
 </figure>
 
+To tweak additional parameters of the AdnMuscle deformer, the following documentation for [AdnRibbonMuscle](ribbon.md) can be consulted.
+
 ## AdnSimshape
 
-To set up an AdnSimshape deformer we will start with a scene with the following elements:
+To create a basic scenario using the AdnSimshape deformer, start with a scene with the following elements:
 
- - An animated facial mesh (to which we will apply the deformer).
+ - An animated facial mesh (to which to apply the deformer).
  - A rest mesh.
+ - A deformation mesh with only the facial animation.
+
+All these meshes must have the same number of vertices and correspond to the same facial model.
 
-All meshes present in the scene have the same number of vertices and correspond to the same facial model.
+<figure>
+  <img src="images/setup_simshape_0.png"> 
+  <figcaption>Figure 25: Basic setup for facial simulations.</figcaption>
+</figure>
 
 ### 1. Creating the deformer
 
-To create the AdnSimshape deformer we will first need to select first the rest mesh and then the animated mesh. In this scenario we will use our animated mesh as our simulated mesh.
+To create the AdnSimshape deformer it is required to select first the rest mesh and then the animated mesh. In this scenario, the animated mesh will be used as the simulated mesh.
+
+Press the ![AdnMuscle](images/adn_simshape.png) shelf button or go to AdonisFX Menu > *Simshape*.
+
+To create the AdnSimshape deformer with some initial customization double click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the "Create" button and the deformer will get created.
```

Comment:
```suggestion
To create the AdnSimshape deformer with some initial customization double-click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the *Create* button and the deformer will get created.
```

Reply:
Done.